Sources/Nitrozen-SwiftUI/Elements/EmptyView/NitrozenEmptyView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  NitrozenEmptyView.swift
+//  
+//
+//  Created by Keyur Patel on 20/04/23.
+//
+
+import SwiftUI
+
+public struct NitrozenEmptyView: View {
+    
+    var topView: AnyView?
+    var title: String
+    var subTitle: String?
+    var buttonTitle: String?
+    var appearance: NitrozenAppearance.EmptyView
+    var onMainButtonClick: ElementTap?
```

Comment:
instead of block, allow develop to pass any type of button here with viewBuilder -- similar concept we have used in Alert